workers/unity/Packages/com.improbable.gdk.core/Config/RuntimeConfig.cs

```diff
@@ -18,12 +18,14 @@ public static class RuntimeConfigDefaults
     /// </summary>
     public static class RuntimeConfigNames
     {
+        public const string DeploymentTag = "deploymentTag";
         public const string LinkProtocol = "linkProtocol";
         public const string LocatorHost = "locatorHost";
         public const string LoginToken = "loginToken";
         public const string ProjectName = "projectName";
         public const string ReceptionistHost = "receptionistHost";
         public const string ReceptionistPort = "receptionistPort";
+        public const string SteamTicket = "steamTicket";
```

Comment:
Why ticket?

Reply:
That's how the thing is called in the worker sdk: https://docs.improbable.io/reference/13.3/csharpsdk/api-reference#improbable-worker-steamcredentials-class